xwiki-platform-core/xwiki-platform-extension/xwiki-platform-extension-security/xwiki-platform-extension-security-index/src/main/java/org/xwiki/extension/security/internal/analyzer/osv/model/response/VulnObject.java

```diff
@@ -154,6 +156,24 @@ public Optional<Version> getMaxFixVersion(Version currentVersion)
             .min(Comparator.naturalOrder());
     }
 
+    /**
+     * @return the list of aliases associated with this vulnerability
+     */
+    public List<String> getAliases()
+    {
+        return this.aliases;
```

Comment:
It would simply a lot of code to return an empty set when `this.alias` is null (i.e. never return null).